resources/views/dimmers.blade.php

```diff
@@ -9,7 +9,7 @@
     $classes = [
         'col-xs-12',
         'col-sm-'.($count >= 2 ? '6' : '12'),
-        'col-md-'.($count >= 3 ? '4' : ($count >= 2 ? '6' : '12')),
+        'col-md-'.($count >= 3 ? '3' : ($count >= 2 ? '6' : '12')),
```

Comment:
This should be calculated base on how many widgets you want to show per row and the default should stay the same as before.